src/from_exception.cpp

```diff
@@ -4,6 +4,147 @@
 // accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
+#if defined(_MSC_VER)
+
+#include <boost/stacktrace/safe_dump_to.hpp>
+#include <windows.h>
+
+extern "C" void** __cdecl __current_exception(); // exported from vcruntime.dll
+#define _pCurrentException static_cast<PEXCEPTION_RECORD>(*__current_exception())
```

Comment:
make it a function ?

Reply:
It is taken from `trnsctrl.h` in vcruntime with slight change. I would like to keep as it is.